addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2159,14 +2171,11 @@ const ListUserValueWidget = UserValueWidget.extend({
      * @private
      */
     _onAddCustomItemClick() {
-        let id;
-        if (this.el.dataset.generateIdForNewItems) {
```

Comment:
Ok, note that removing parameters like that is not something to be done in stable in general. I'll choose to make an exception here as it is an option to a very specific internal widget introduced in 15.2 for your snippet that we review now.